query/src/test/scala/hmda/query/projections/institutions/InstitutionViewSpec.scala

```diff
@@ -0,0 +1,57 @@
+package hmda.query.projections.institutions
+
+import akka.testkit.TestProbe
+import hmda.model.institution.InstitutionGenerators
+import hmda.persistence.messages.CommonMessages.GetState
+import hmda.persistence.messages.events.institutions.InstitutionEvents.{ InstitutionCreated, InstitutionModified }
+import hmda.persistence.model.ActorSpec
+import hmda.persistence.processing.HmdaQuery.EventWithSeqNr
+import hmda.query.projections.institutions.InstitutionView._
+
+class InstitutionViewSpec extends ActorSpec {
+
+  val i1 = InstitutionGenerators.institutionGen.sample.get
+  val i2 = InstitutionGenerators.institutionGen.sample.get
+  val i3 = InstitutionGenerators.institutionGen.sample.get
+  val i4 = i3.copy(cra = true)
+
+  val institutionQuery = createInstitutionQuery(system)
+
+  val probe = TestProbe()
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    institutionQuery ! EventWithSeqNr(1, InstitutionCreated(i1))
+    institutionQuery ! EventWithSeqNr(2, InstitutionCreated(i2))
+    institutionQuery ! EventWithSeqNr(3, InstitutionCreated(i3))
+    institutionQuery ! EventWithSeqNr(4, InstitutionModified(i4))
+  }
+
+  "Institutions Projection" must {
```

Comment:
I think that it would be useful to have a test that `InstitutionsView` will successfully read events that it doesn't have from the `InstitutionPersistence` journal, though I don't know if this is the right place for it as that seems like more of an integration test.